packages/mural-picker/src/components/mural-create/index.tsx

```diff
@@ -16,6 +16,7 @@ import { getCommonTrackingProperties } from '../../common/tracking-properties';
 import { CardListSection } from '../card-list/card-list-section';
 import { ErrorHandler } from '../types';
 
+import { getApiError } from '../../../../mural-client/src';
```

Comment:
I don't know why I missed that, but I think this line is the issue

Should be:
```
import {
  getApiError,
} from '@muraldevkit/mural-integrations-mural-client';
```

Reply:
Oh!! Thank you!